go/worker/keymanager/api/api.go

```diff
@@ -0,0 +1,100 @@
+package api
+
+import (
+	"fmt"
+
+	core "github.com/libp2p/go-libp2p-core"
+
+	"github.com/oasisprotocol/oasis-core/go/common"
+)
+
+// StatusState is the concise status state of the key manager worker.
+type StatusState uint8
+
+const (
+	// StatusStateReady is the ready status state.
+	StatusStateReady StatusState = iota
```

Comment:
Given that these are serialized please use explicit values instead of relying on `iota` to ensure the encoding remains stable.